src/lib/rmt/wizard_scc_page.rb

```diff
@@ -127,7 +127,17 @@ def scc_credentials_valid?
     )
 
     uri = URI('https://scc.suse.com/connect/organizations/systems')
-    req = Net::HTTP::Get.new(uri)
+    if @config['proxy'] and @config['proxy']['proxy']
+      p_uri = URI(@config['proxy']['proxy'])
+      p_user = p_uri.user.empty? ? @config['proxy']['proxy_user'] : p_uri.user
+      p_pass = p_uri.password.empty? ? @config['proxy']['proxy_password'] : p_uri.user
```

Comment:
Is this supposed to take the URI user if the URI password is non-empty?

Reply:
oops. thank you for spotting that

Reply:
i have revised the commit to match more closely with how rmt behaves here. rmt does not consider username or password from the host URI.